src/class-parsely.php

```diff
@@ -186,7 +186,15 @@ public function insert_page_header_metadata(): void {
 
 		// Assign default values for LD+JSON
 		// TODO: Mapping of an install's post types to Parse.ly post types (namely page/post).
-		$parsely_page = $this->construct_parsely_metadata( $parsely_options, $post );
+		if ( is_int( $post ) ) {
+			$parsed_post = WP_Post::get_instance( $post );
+			if ( $parsed_post === false ) {
+				return;
+			}
+			$parsely_page = $this->construct_parsely_metadata( $parsely_options, $parsed_post );
+		} else {
+			$parsely_page = $this->construct_parsely_metadata( $parsely_options, $post );
+		}
```

Comment:
I wonder if this would suffice:

```suggestion
		$parsely_page = $this->construct_parsely_metadata( $parsely_options, get_post( $post ) );
```